samples/Localization.StarterWeb/Localization.StarterWeb.csproj

```diff
@@ -42,7 +34,6 @@
 
   <ItemGroup>
     <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="1.2.0-*" />
-    <DotNetCliToolReference Include="Microsoft.Extensions.SecretManager.Tools" Version="1.0.1-*" />
```

Comment:
> No issue occurs during dotnet restore.

That means restore on `Microsoft.Extensions.SecretManager.Tools` is working. If we were to change this package to target `netcoreapp2.0`, this would cause restore to fail because dotnet-CLI does not (yet) support tools packages that target .NET Core 2.0.